docs/docs/software-architecture.md

```diff
@@ -75,29 +74,29 @@ The HTLC contains several sub-components. A crucial part here, is the secret and
 
 Verification:   
 The Verification contains the signature verification sub-component which is consulted to verify the swap partners’ signatures. This sub-component comprises of a sign-transaction interface which locally compares the private key to the public key in the browser. If the verification is successful, the swap is executed. If not, the verification fails, and the refund process is initiated.
-
+   
 ### 3.1. UML Diagram
 
 ![](img/UML_new.svg)   
 Figure 2: UML Diagram (Swapchain, 2020) [4]
 
-
+   
 ## 4. Use Case
 
 The use case diagram is used to visualize the Swapchain application and its actors.
-
+   
 ### 4.1. Use Case Diagram
 
 ![](img/newSW_Architecture.svg)     
 Figure 3: Use Case Diagram (Swapchain, 2020) [4]
-
+   
 ### 4.2. Use Case Description
 
 | Use Case Name           | Scenario                               | Triggering Event                   | Actors            | Related Use Case | Preconditions                                                                                                                                       | Post Conditions                                                     | Flow of Events                                                                                                                                                                                                           | Exception Conditions                                          |
 | ----------------------- | -------------------------------------- | ---------------------------------- | ----------------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------- |
 | Atomic cross-chain swap | Two users want to exchange BTC and BTS | The user submits an exchange order | User 1 and User 2 | None             | - Users should be in possession of the cryptocurrency that is desired by the other party     - Users should open a HTLC in their respective Blockchains | - Validating the user     - Hash and time lock conditions should be met | - Submit desired order     - Start a HTLC     - Fund and redeem a swap     - Key pair generator     - Verification process of validity between private and public keys     - Transaction successful or funds are refunded | - Network failure   - App crashing   - Too much market volatility |
```

Comment:
delete: the users submits an exchange order because we won't have an orderbook running at the time of the end of the project